### ISSUE TITLE
Installation struck on Starting Watchtower OK

### ISSUE THREAD

**Issue by kdpuvvadi** [Jigsaw-Code/outline-server#25]
Using Azure VM, Ubuntu 16.04 LTS. New VM nothing installed except SSH-server. Copy pasted the installing script from Manager app and started downloading files and installed docker and now it been struck here at `> Starting Watchtower ......................... OK`. 
It's been an hour since it started

Edit: Same issue even after setting the public ip

**Comment by yuzhe922** [comment]
Same issues

**Comment by 7sDream** [comment]
dup of #22 ?

**Comment by kdpuvvadi** [comment]
Similar, he got it fixed it by setting public IP of the server but here it is working with that fix

**Comment by xunxuntj** [comment]
I tried to export the IP but it didn't help

**Comment by samyules** [comment]
Same issue on Ubuntu 17.10.

**Comment by pcause** [comment]
same on ubuntu 18.10 beta

**Comment by BlueskyFR** [comment]
Same on Ubuntu 17.10... Any fix?

**Comment by zl1775** [comment]
Hi I'm the OP of #22, although I didn't get the issue fixed by changing to public IP, I think you should take a look at #10, someone with the exact same set up as you on Azure got the issue fixed by changing Inbound - Outbound rules for ports 1024-65535. I suspect the issue of being stuck on "Starting Watchtower" has to do with ports being behind some form of a firewall.

**Comment by kdpuvvadi** [comment]
Looks like firewall is the main problem here. I've open the ports and it's got installed. I think user should have option to choose the port at least.

**Comment by fortuna** [comment]
This PR should help with better communication on what's going on:
https://github.com/Jigsaw-Code/outline-server/pull/38

It also tells you what ports to open

**Comment by fortuna** [comment]
The original issue was likely due to a firewall. https://github.com/Jigsaw-Code/outline-server/pull/38 should have clarified what's going on in the installation and provide the information needed for you to unblock the ports.

I'm marking this as closed. Feel free to open a new issue for things that were not resolved with the change.
